datafusion/physical-expr/src/intervals/interval_aritmetic.rs

```diff
@@ -692,11 +709,22 @@ fn next_value<const INC: bool>(value: ScalarValue) -> ScalarValue {
 }
 
 /// This function computes the cardinality ratio of the given intervals.
+/// If it cannot be calculated, it returns 1.0 meaning full selective.
```

Comment:
```suggestion
/// This function computes the selectivity by computing the cardinality ratio of the given intervals.
/// If this can not be calculated for some reasons, returns `1.0` meaning full selective / no filtering.
```